src/cryptonote_core/cryptonote_core.h

```diff
@@ -849,7 +849,7 @@ namespace cryptonote
       *
       * @param keys The container in which to return the keys
       */
-     void get_all_service_node_public_keys(std::vector<crypto::public_key>& keys) const;
+     void get_all_service_nodes_public_keys(std::vector<crypto::public_key>& keys) const;
```

Comment:
Reason for the change?

Reply:
Consistency with the other service node api